src/utilities.jl

```diff
@@ -17,6 +17,7 @@ using the trapezoidal rule.
 """
 function area_under_curve(x, y)
     @assert length(x) == length(y)
+    length(x) == 0 && return 0.0
```

Comment:
hit some strange edge case in https://github.com/beacon-biosignals/LighthouseFlux.jl/pull/20 which is why I added this; possibly this should error or something instead though

Reply:
In any case, let's add a unit tests that demonstrates the expected behavior.